doc/developers/contributing.rst

```diff
@@ -23,6 +23,9 @@ See :ref:`new_contributors` to get started.
     discussions.
 
     We aspire to treat everybody equally, and value their contributions.
+    We are particularly seeking people from underrepresented backgrounds
+    in Open Source Software and this project to participate and contribute
```

Comment:
Nit?

The "and this project" makes this seem a little unclear. How about a slight modification:

> We are particularly seeking from underrepresented backgrounds in Open Source Software to participate and contribute their expertise and experience to this project.

Reply:
I agree this would be easier to read. But I feel it's different to say we want good representation in open source and good representation in this project? Though I don't have a strong opinion.

Reply:
Using `scikit-learn` directly reads a more clearer?

> We are particularly seeking people from underrepresented backgrounds in Open Source Software and scikit-learn to participate and contribute